### PR TITLE
Add breakers and collectors for Abuse.ch and Alienvault

### PR DESCRIPTION
**Pull request by mulibu-flyingk**
Hi Brendan,

I have added two Collectors (including breakers) for threat intel feeds from Abuse.ch and Alienvault.

Cheers
Christoph

### CODE REVIEW

File: collectors/rest/abuse-ch/collector.json [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+      "logLevel": "info",
+      "jobTimeout": "0",
+      "mode": "run",
+      "timeRangeType": "relative",
+      "timestampTimezone": "UTC",
+      "expression": "true",
+      "minTaskSize": "1MB",
+      "maxTaskSize": "10MB"
+    },
+    "enabled": true
```
Comment (bdalpe) [inline collectors/rest/abuse-ch/collector.json:22]: Can you make this disabled please? Don't want customers to unsuspectingly start running a collector job without explicit instruction.

File: collectors/rest/alienvault/collector.json [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+      "logLevel": "info",
+      "jobTimeout": "0",
+      "mode": "run",
+      "timeRangeType": "relative",
+      "timestampTimezone": "UTC",
+      "expression": "true",
+      "minTaskSize": "1MB",
+      "maxTaskSize": "10MB"
+    },
+    "enabled": true
```
Comment (bdalpe) [inline collectors/rest/alienvault/collector.json:22]: Same as the other disabled comment

**Review by bdalpe — CHANGES_REQUESTED** [review]
Thanks @mulibu-flyingk! Just two quick changes please 😄

File: collectors/rest/abuse-ch/collector.json (hunk 1 above)
Comment (mulibu-flyingk) [reply collectors/rest/abuse-ch/collector.json]: Hi @bdalpe, I have disabled the schedule and updated the README accordingly.

File: collectors/rest/alienvault/collector.json (hunk 2 above)
Comment (mulibu-flyingk) [reply collectors/rest/alienvault/collector.json]: Hi @bdalpe, I have disabled the schedule and updated the README accordingly.

**Review by bdalpe — APPROVED** [review]

**Pull request merged**